pypeit/core/wavecal/autoid.py

```diff
@@ -891,7 +893,8 @@ def full_template(spec, lamps, par, ok_mask, det, binspectral, nsnippet=2, debug
 
     # Load template
     if template_dict is None:
-        temp_wv, temp_spec, temp_bin = waveio.load_template(par['reid_arxiv'], det)
+        temp_wv, temp_spec, temp_bin = waveio.load_template(
+            par['reid_arxiv'], det, mnmx=par['mnxwv_arxiv'])
```

Comment:
Can we change these to `range`, `waverange`, or `wvrng`?  `mnmx` and `mnxwv` feel too obscure.

Reply:
now using `wvrng`.  Will test